### PR TITLE
Fix rendering issue in onoff switch. Apply guidelines.

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1034
[Problem] On/off switch has rendering issue
[Solution] Adjust on/off switch to the UX guideline

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
New look after PR
image
image

File: src/css/profile/wearable/changeable/theme-circle/toggleswitch.less [hunk 1]
```diff
@@ -81,11 +82,11 @@
 			position: absolute;
 			content: '';
 			transform: scale(1);
-			height: 51 * @unit_base;
+			height: 52 * @unit_base;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/theme-circle/toggleswitch.less]: You have to decide which unit you use

Comment (lslachciak) [reply src/css/profile/wearable/changeable/theme-circle/toggleswitch.less]: decided px_base
guide is in px

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**